base_layer/wallet_ffi/README.md

```diff
@@ -35,11 +35,23 @@ For macOS Mojave additional headers need to be installed, run
 ```Shell Script
 open /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg
 ```
-and follow the prompts
+and follow the prompts.
+
+For Catalina, if you get compilation errors such as these:
+
+    xcrun: error: SDK "iphoneos" cannot be located 
+    xcrun: error: unable to lookup item 'Path' in SDK 'iphoneos'
+
+Switch the XCode app defaults with:
+
+    sudo xcode-select --switch /Applications/Xcode.app
```

Comment:
If this command fails, XCode was not found and needs to be installed/re-installed.